config/owasp/suppressions.xml

```diff
@@ -26,5 +26,7 @@
    ]]></notes>
     <cve>CVE-2021-28170</cve>
     <cve>CVE-2021-22119</cve>
+    <cve>CVE-2021-22096</cve>
```

Comment:
to be upgraded as a part of the spring web version story we have, so can suppress it until that is done